packages/tools/src/eventListeners/mouse/mouseDoubleClickListener.ts

```diff
@@ -35,7 +35,21 @@ function mouseDoubleClickListener(evt: MouseEvent): void {
     deltaPoints,
   };
 
-  triggerEvent(element, Events.MOUSE_DOUBLE_CLICK, eventDetail);
+  const preventDefault = !triggerEvent(
```

Comment:
can we rename this to `consumed` or similar?

Reply:
or `eventDidNotPropagate`

Reply:
or even better `const eventDidPropagate = trigger ...`

Reply:
Note that `triggerEvent` in turn calls `dispatchEvent`. I checked the mdn docs for `dispatchEvent` (https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/dispatchEvent) and its return value has NOTHING to do with propagation. In fact it is related to `preventDefault`. So I still prefer the name I had. However of your choices I think `consumed` is best.